d_calculate_handoff_coefficients/src/collate_handoff_coefficients.R

```diff
@@ -10,14 +10,14 @@
 #' 
 collate_handoff_coefficients <- function() {
   #list the files in coefficients
-  files <- list.files("c_calculate_handoff_coefficients/mid",
+  files <- list.files("cdcalculate_handoff_coefficients/mid",
```

Comment:
```suggestion
  files <- list.files("d_calculate_handoff_coefficients/mid",
```

Reply:
got one! 😆

Reply:
BAHAHAH that one is egregious. THANK YOU